i18n/de.json

```diff
@@ -1400,6 +1400,14 @@
     "translation": "Anfragenbegrenzereinstellung nicht korrekt konfiguriert, verwende VaryByHeader und deaktiviere VaryByRemoteAddr"
   },
   {
+    "id": "api.server.start_server.rate_limiting_memory_store",
+    "translation": "Konnte Anfragenbegrenzer nicht initialisieren."
```

Comment:
Anfragenbegrenzer sounds very "try hard". In my opinion there are certain things like error messages (or single terms in those messages) that better stay English.